internal/lib/sandbox/namespaces_linux.go

```diff
@@ -152,6 +158,14 @@ func pinNamespaces(nsTypes []NSType, cfg *config.Config, idMappings *idtools.IDM
 	return returnedNamespaces, nil
 }
 
+func getSysctlForPinns(sysctls map[string]string) string {
+	g := new(bytes.Buffer)
+	for key, value := range sysctls {
+		fmt.Fprintf(g, "'%s=%s':", key, value)
+	}
+	return strings.TrimSuffix(g.String(), ":")
```

Comment:
Do we assume that there are no values with `:` or `=` in them? Should we actually check it (not here, but maybe when parsing annotations).

Reply:
that's a great point. a quick `sudo sysctl -a | fold -w1 | sort -u` shows me some with `:`, though `=` is a special character in sysctl land, so that should be allowed
I've opted to deliminate them with `+` (not a special bash character, not included in the characters of sysctls on my machine). No spot seems right to find `+` though, we never traverse the whole string, mostly pass it from place to place